doc/Makefile

```diff
@@ -4,176 +4,182 @@
+	doc/askrene-unreserve.7 \
+	doc/autoclean-once.7 \
+	doc/autoclean-status.7 \
+	doc/batching.7 \
+	doc/bkpr-channelsapy.7 \
+	doc/bkpr-dumpincomecsv.7 \
+	doc/bkpr-editdescriptionbyoutpoint.7 \
+	doc/bkpr-editdescriptionbypaymentid.7 \
+	doc/bkpr-inspect.7 \
+	doc/bkpr-listaccountevents.7 \
+	doc/bkpr-listbalances.7 \
+	doc/bkpr-listincome.7 \
+	doc/blacklistrune.7 \
+	doc/check.7 \
+	doc/checkmessage.7 \
+	doc/checkrune.7 \
+	doc/close.7 \
+	doc/commando-blacklist.7 \
+	doc/commando.7 \
+	doc/commando-listrunes.7 \
+	doc/commando-rune.7 \
+	doc/connect.7 \
+	doc/createinvoice.7 \
+	doc/createonion.7 \
+	doc/createrune.7 \
+	doc/datastore.7 \
+	doc/datastoreusage.7 \
+	doc/decode.7 \
+	doc/decodepay.7 \
+	doc/deldatastore.7 \
+	doc/delforward.7 \
+	doc/delinvoice.7 \
+	doc/delpay.7 \
+	doc/deprecations.7 \
+	doc/dev-forget-channel.7 \
+	doc/dev-splice.7 \
+	doc/disableinvoicerequest.7 \
+	doc/disableoffer.7 \
+	doc/disconnect.7 \
+	doc/emergencyrecover.7 \
+	doc/enableoffer.7	\
+	doc/exposesecret.7 \
+	doc/feerates.7 \
+	doc/fetchinvoice.7 \
+	doc/fundchannel_cancel.7 \
+	doc/fundchannel_complete.7 \
+	doc/fundchannel.7 \
+	doc/fundchannel_start.7 \
+	doc/funderupdate.7 \
+	doc/fundpsbt.7 \
+	doc/getemergencyrecoverdata.7 \
+	doc/getinfo.7 \
+	doc/getlog.7 \
+	doc/getroute.7 \
+	doc/getroutes.7 \
+	doc/help.7 \
+	doc/injectonionmessage.7 \
+	doc/injectpaymentonion.7 \
+	doc/invoice.7 \
+	doc/invoicerequest.7 \
+	doc/keysend.7 \
+	doc/listaddresses.7 \
+	doc/listchannels.7 \
+	doc/listclosedchannels.7 \
+	doc/listconfigs.7 \
+	doc/listdatastore.7 \
+	doc/listforwards.7 \
+	doc/listfunds.7 \
+	doc/listhtlcs.7 \
+	doc/listinvoicerequests.7 \
+	doc/listinvoices.7 \
+	doc/listnodes.7 \
+	doc/listoffers.7 \
+	doc/listpays.7 \
+	doc/listpeerchannels.7 \
+	doc/listpeers.7 \
+	doc/listsendpays.7 \
+	doc/listtransactions.7 \
+	doc/makesecret.7 \
+	doc/multifundchannel.7 \
+	doc/multiwithdraw.7 \
+	doc/newaddr.7 \
+	doc/notifications.7 \
+	doc/offer.7 \
+	doc/openchannel_abort.7 \
+	doc/openchannel_bump.7 \
+	doc/openchannel_init.7 \
+	doc/openchannel_signed.7 \
+	doc/openchannel_update.7 \
+	doc/parsefeerate.7 \
+	doc/pay.7 \
+	doc/ping.7 \
+	doc/plugin.7 \
+	doc/preapproveinvoice.7 \
+	doc/preapprovekeysend.7 \
+	doc/reckless.7 \
+	doc/recoverchannel.7 \
+	doc/recover.7 \
+	doc/renepay.7 \
+	doc/renepaystatus.7 \
+	doc/reserveinputs.7 \
+	doc/sendcustommsg.7 \
+	doc/sendinvoice.7 \
+	doc/sendonion.7 \
+	doc/sendpay.7 \
+	doc/sendpsbt.7 \
+	doc/setchannel.7 \
+	doc/setconfig.7 \
+	doc/setpsbtversion.7 \
+	doc/showrunes.7 \
+	doc/signinvoice.7 \
+	doc/signmessage.7 \
+	doc/signpsbt.7 \
+	doc/splice_init.7 \
+	doc/splice_signed.7 \
+	doc/splice_update.7 \
+	doc/staticbackup.7 \
+	doc/stop.7 \
+	doc/txdiscard.7 \
+	doc/txprepare.7 \
+	doc/txsend.7 \
+	doc/unreserveinputs.7 \
+	doc/upgradewallet.7 \
+	doc/utxopsbt.7 \
+	doc/waitanyinvoice.7 \
+	doc/waitblockheight.7 \
+	doc/waitinvoice.7 \
+	doc/wait.7 \
+	doc/waitsendpay.7 \
+	doc/withdraw.7 \
+	doc/xpay.7
```

Comment:
The .7 pages still need the lightning- prefix, as otherwise names are too general, unfortunately.

Reply:
This is the list of markdown pages (*.md). We are later prefixing `lightning-` to this list for generating MANPAGES.